src/main/java/com/sap/cloudfoundry/client/facade/adapters/CloudFoundryClientFactory.java

```diff
@@ -165,4 +169,15 @@ private reactor.netty.http.client.HttpClient getAdditionalHttpClientConfiguratio
         return clientWithOptions;
     }
 
+    private static WebClient buildWebClient() {
```

Comment:
I think the main reason for choosing the jdk http client is that the direct memory was abnormally high when using reactor netty. Does the spring client cause the same issue?

Reply:
That's correct when using DataBuffers (ByteBuffers) to convert the response to InputStream. (Reproduced)
Parsing it from the response does not use direct memory. In any case, we're reading the whole stream to parse the response